frontend/src/components/layouts/ImageGrid.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import styles from "./ImageGrid.module.scss";
+
+interface GridProps {
+  children: React.ReactChild | React.ReactChild[];
+  className?: string;
+}
+
+export function Grid({ children, className }: GridProps) {
+  return (
+    <div className={`${styles.grid} ${className}`}>
+      {React.Children.map(children, (child) => (
+        <div className={styles.gridItem}>{child}</div>
+      ))}
```

Comment:
```suggestion
      {React.Children.map(children, (element) => 
        React.cloneElement(element, { className: `${styles.gridItem} ${element.props.className || ''}`)
      )}
```
I would suggest doing this to prevent creating two grid item divs for one item. In this way, the class name is simply merged with the class name that is already provided to the child component